GVFS/GVFS.Common/Maintenance/PrefetchStep.cs

```diff
@@ -151,8 +153,13 @@ private bool TryGetMaxGoodPrefetchTimestamp(out long maxGoodTimestamp, out strin
             maxGoodTimestamp = -1;
 
             int firstBadPack = -1;
-            for (int i = 0; !this.Stopping && i < orderedPacks.Count; ++i)
+            for (int i = 0; i < orderedPacks.Count; ++i)
             {
+                if (this.Stopping)

```

Comment:
Do we need this check here?  

If we're stopping `this.RunGitCommand` (on line 172) will throw a `StoppingException`, and if we skip that call to `RunGitCommand` we'll throw a `StoppingException` on line 198.